tests/integration/cairo1_compiler_bindings/tests_collector_and_compiler_test/test_with_contract_deps.cairo

```diff
@@ -0,0 +1,7 @@
+use external_contract::contract::SampleContract::InnerStruct;
+
+#[test]
+fn test_importing_from_contract(){
```

Comment:
you never really run this test or did I miss something?

Reply:
right i don't, since it should check if it's possible to do so. if it wouldn't be, it'd crash in collecting.